### PR TITLE
chart: responsive chart example

### PR DESCRIPTION
**Pull request by marin-bratanov**

### CODE REVIEW

File: chart/responsive-chart/Shared/IndividualChart.razor [hunk 1]
```diff
@@ -0,0 +1,79 @@
+﻿@using ResponsiveChart.Data
+@implements IDisposable
+
+@code {
+    TelerikChart theChart { get; set; } // you need references to the charts you need to resize
```
Comment (Stamo-Gochev) [inline chart/responsive-chart/Shared/IndividualChart.razor]: Using `Chart` seems like a better name to me.

File: chart/responsive-chart/Pages/Index.razor [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    .div50 {
+        width: 50%;
+        float: left;
+    }
+</style>
+
+Resize the browser window to see the charts respond to the size change
+
+<div class="div100">
+    <IndividualChart />
```
Comment (Stamo-Gochev) [inline chart/responsive-chart/Pages/Index.razor]: Why not something like `ResponsiveChart` instead?

File: chart/responsive-chart/wwwroot/windowResizeHandler.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿function raiseResizeEvent() {
+    var namespace = 'ResponsiveChart'; // the namespace of the app, you will have to change this for your app
+    var method = 'RaiseWindowResizeEvent'; //the name of the method in our "service"
+    DotNet.invokeMethodAsync(namespace, method, window.innerWidth, window.innerHeight);
+}
+
+//throttle resize event, taken from https://stackoverflow.com/a/668185/812369
+var TO = false;
+window.addEventListener("resize", function () {
+    if (TO !== false)
```
Comment (Stamo-Gochev) [inline chart/responsive-chart/wwwroot/windowResizeHandler.js]: Can be renamed to `timeout`.

File: chart/responsive-chart/Pages/Index.razor (hunk 2 above)
Comment (marin-bratanov) [reply chart/responsive-chart/Pages/Index.razor]: It would clash with the project name. I renamed to `ResponsiveChartComponent`

**Pull request merged**